.github/workflows/llm_performance_tests.yml

```diff
@@ -90,6 +91,10 @@ jobs:
       - name: Test on xpu(transformers==4.31.0)
         shell: bash
         run: |
+          pip uninstall -y bigdl-llm
+          pip install --pre --upgrade ipex-llm
+          find "python/llm" -type f -name "*.py" -exec sed -i '1s/^/import ipex_llm\n/' {} +
```

Comment:
the goal of compatibility test is to run original bigdl code without `import ipex_llm`